rubocop.yml

```diff
@@ -173,3 +173,29 @@ Layout/FirstHashElementIndentation:
   # This one in particular forbids what Emacs does by default for multi-line
   # hash entries
   Enabled: false
+
+# Pending Cops as of 0.87.1
+Layout/EmptyLinesAroundAttributeAccessor:
+  Enabled: true
+Layout/SpaceAroundMethodCallOperator:
+  Enabled: true
+Lint/DeprecatedOpenSSLConstant:
+  Enabled: true
+Lint/MixedRegexpCaptureTypes:
+  Enabled: true
+Style/AccessorGrouping: # maybe
+  Enabled: true
+Style/BisectedAttrAccessor:
+  Enabled: true
+Style/ExponentialNotation:
+  Enabled: true
+Style/RedundantAssignment:
```

Comment:
This one I expected to have many violations, but I only saw one and it didn't seem to have any justification.